examples/data-transfer-pg/main.py

```diff
@@ -62,41 +69,52 @@ def connect_to_databases():
             os.environ['DB2_USERNAME'], 
             os.environ['DB2_PASSWORD']
         )
+
     return source_db_conn, dest_db_conn
 
 
+def populate_data(count: int, db_cursor):
+    ''' Generate user data. '''
+    fake = Faker()
+    row = lambda n: (n + 1, repr(fake.name()), repr(fake.city()))
+    
+    for i in range(count):
+        db_cursor.execute(SQL_INSERT_DATA, row(i))
+
+
 def extract_sql(file: str):
     ''' Reads an SQL file and returns it's contents.'''
     with open (file, 'rt') as file:
         contents = file.read()  
+
     return contents
 
 
 def transfer_data(source_db_cursor, dest_db_cursor, dest_db_conn):
-    ''' Extracts fruits data from source database and stores them in destination database.'''
-    print(f'Extracting fruits data from source database.')
-    source_db_cursor.execute('SELECT * FROM fruits')
+    ''' Extracts users data from source database and stores them in destination database.'''
+    print(f'Extracting users data from source database.')
+    source_db_cursor.execute('SELECT * FROM users')
     rows = source_db_cursor.fetchall()
 
-    print(f'Transferring fruits data to destination database.')
+    print(f'Transferring users data to destination database.')
     for row in rows:
-        dest_db_cursor.execute('INSERT INTO fruits VALUES (?, ?, ?)', (row.id, row.name, row.quantity))
+        dest_db_cursor.execute('INSERT INTO users VALUES (?, ?, ?)', (row.id, row.name, row.city))
```

Comment:
let's use the existing constant `SQL_INSERT_DATA` here too.

Reply:
Done